libs/util.js

```diff
@@ -14,6 +14,7 @@ exports.isLoggedIn = function (req, res, next) {
 
   // if they aren't redirect them to the home page
   res.redirect('/');
+  return;
```

Comment:
return with no value at the end of function does not make sense. Please change the function to always return no value. for example:
```
 if (req.isAuthenticated()) {
    return next();
  }
```
===>
 ```
 if (req.isAuthenticated()) {
    next();
    return
  }
```

Reply:
fixed